src/schemas/AlertThresholdObject.yaml

```diff
@@ -0,0 +1,13 @@
+allOf:
+  - $ref: "./AlertThresholdBaseObject.yaml"
+type: object
```

Comment:
Same comment regarding `allOf`

Reply:
Thank you! It's aligned in the official docs indeed